bundles/paikkatietoikkuna/coordinateconversion/instance.js

```diff
@@ -0,0 +1,186 @@
+/*
+* Oskari.app.playBundle(
+    {
+    bundlename : 'coordinateconversion',
+        metadata : {
+            "Import-Bundle" : {
+                "coordinateconversion" : {
+                    bundlePath : '/Oskari/packages/paikkatietoikkuna/bundle/'
+                }
+            }
+        }
+});
+            var l = appSetup.startupSequence.length;
+            appSetup.startupSequence[l] = {
+                "bundleinstancename":"coordinateconversion",
+                "bundlename":"coordinateconversion" ,
+                "fi":"",
+                "sv":"",
+                "en":"",
+                "title":"coordinateconversion" 
+            }
+            appSetup.startupSequence[l].metadata= { "Import-Bundle": { "coordinateconversion": { "bundlePath": "/Oskari/packages/paikkatietoikkuna/bundle/" } } };
```

Comment:
Maybe add some comments what these are. Also startupSequence.push() would be more readable. You don't need fi, sv, en, title, bundleinstancename here.